eslintrc.js

```diff
@@ -0,0 +1,118 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+    'jest/globals': true,
+  },
+  extends: [
+    // 'eslint:recommended',
+    // 'plugin:import/recommended',
+    // 'plugin:react/recommended',
+    // 'plugin:@typescript-eslint/eslint-recommended',
+    // 'plugin:@typescript-eslint/recommended',
+    'prettier',
+    'prettier/react',
+    'prettier/@typescript-eslint',
+  ],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    project: ['tsconfig.json', 'cypress/tsconfig.json'],
+    sourceType: 'module',
+  },
+  settings: {
+    react: {
+      version: 'detect',
+    },
+  },
+  plugins: [
+    '@typescript-eslint',
+    '@typescript-eslint/tslint',
+    'jest',
+    'react',
+    'prettier',
+    'react-hooks',
+  ],
+  rules: {
+    '@typescript-eslint/class-name-casing': 'error',
+    '@typescript-eslint/indent': 'off',
+    '@typescript-eslint/interface-name-prefix': 'error',
+    '@typescript-eslint/member-delimiter-style': 'off',
+    '@typescript-eslint/type-annotation-spacing': 'off',
+    'arrow-parens': ['off', 'as-needed'],
+    'default-case': 'error',
+    'dot-notation': 'error',
+    'eol-last': 'off',
+    'guard-for-in': 'error',
+    'linebreak-style': 'off',
+    'new-parens': 'off',
+    'newline-per-chained-call': 'off',
+    'no-caller': 'error',
+    'no-debugger': 'error',
+    'no-empty': 'error',
+    'no-empty-function': 'error',
+    'no-eval': 'error',
+    'no-extra-semi': 'off',
+    'no-fallthrough': 'error',
+    'no-irregular-whitespace': 'off',
+    'no-multiple-empty-lines': 'off',
+    'no-new-wrappers': 'error',
+    'no-param-reassign': 'error',
+    'no-undef-init': 'error',
+    'no-unused-labels': 'error',
+    'no-use-before-define': 'warn', // Decide on this
+    'no-var': 'error',
+    'prefer-const': 'error',
+    'quote-props': 'off',
+    'space-before-function-paren': 'off',
+    'react/jsx-no-bind': 'warn', // Update
+    curly: 'error',
+    radix: 'error',
+    'prettier/prettier': [
+      'error',
+      {
+        arrowParens: 'avoid',
```

Comment:
These are all duplicated from `.prettierrc`. Are these not automatically inferred? If not, maybe we could just require that file and reference the config here?

Something like:
```ts
const fs = require('fs');
const prettierConfig = JSON.parse(fs.readFileSync('.prettierrc'))

/// ...

'prettier/prettier': ['error', prettierConfig]
```

Reply:
Or maybe better, change `.prettierrc` to `.prettierrc.js` and just `require` it